### DISCUSSION TITLE
error with running "cast wallet import"

### DISCUSSION THREAD

**Discussion by Imad-Allal** [Cyfrin/foundry-full-course-cu#1219] [Q&A]
Hey guys,

So apparently there is a new way to use private keys in our projects thanks to the ERC2335, but it's not working for me, and my problem seems to be that the "import" keyword doesn't exist (as shown bellow). I tried to update forge (which is on 0.2.0) but nothing happend...
Here are some screenshots
![Capture d'écran 2023-12-21 073800](https://github.com/Cyfrin/foundry-full-course-f23/assets/80546947/fea53553-724e-4334-96b3-2a21d8047633)
![Capture d'écran 2023-12-21 073815](https://github.com/Cyfrin/foundry-full-course-f23/assets/80546947/9572960c-a562-405f-b721-926c3d2f1660)
![Capture d'écran 2023-12-21 073827](https://github.com/Cyfrin/foundry-full-course-f23/assets/80546947/2d811cf0-b375-4c07-961d-a40b55073345)
![Capture d'écran 2023-12-21 073840](https://github.com/Cyfrin/foundry-full-course-f23/assets/80546947/fa893a20-53cb-4b80-a3f5-6d43bb55e68b)

**Comment by Jahanzeb9999** [comment]
did you get it? I am stuck on this

**Reply by Imad-Allal** [reply]
Still nothing :(

**Comment by korchynskyi** [comment]
Hi guys! Try to reinstall Foundry.
New way to use private keys works fine for me with Foundry install on 23.12.2023
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/13314693/033a221e-6bfc-4a02-bc2b-830f378c8b57)

**Reply by Jahanzeb9999** [reply]
I already have installed, but still there is error with import
